modules/apps/item-selector/item-selector-web/src/test/java/com/liferay/item/selector/web/internal/ItemSelectorImplTest.java

```diff
@@ -106,6 +107,23 @@ public static void setUpClass() {
 		portalUtil.setPortal(new PortalImpl());
 	}
 
+	@After
+	public void tearDown() {
+		if ((_flickrItemSelectorCriterionHandlerServiceRegistration != null) ||
```

Comment:
@W2jx Here the 2 null checks should not be put together. What if one of the registration is null, while the other is not? You will still get NPE in the if-body.

You should deactivate the `_itemSelectorImpl`, then null-check each registration and unregister it.